src/meta/meta_ckpt.c

```diff
@@ -1044,19 +1044,19 @@ __wt_meta_sysinfo_set(WT_SESSION_IMPL *session)
     }
 
     /* Record snapshot information in metadata for checkpoint. */
-    if (txn->snapshot_count > 0) {
-        WT_ERR(__wt_buf_fmt(session, buf,
-          WT_SYSTEM_CKPT_SNAPSHOT_MIN "=%" PRIu64 "," WT_SYSTEM_CKPT_SNAPSHOT_MAX "=%" PRIu64
-                                      "," WT_SYSTEM_CKPT_SNAPSHOT_COUNT "=%" PRIu32
-                                      "," WT_SYSTEM_CKPT_SNAPSHOT "=[",
-          txn->snap_min, txn->snap_max, txn->snapshot_count));
+    WT_ERR(__wt_buf_fmt(session, buf,
```

Comment:
Why is this changing?

Reply:
Earlier the code used to store the recovered checkpoint snapshot data to metadata only when the snapshot_count was 0, but we figured out some visibility issue while running `schma_abort` and `random_abort` tests. So, now we will store the recovery checkpoint snapshot min snapshot max irrespective of snapshot count.